### ISSUE TITLE
could you give a example about how to use selu rightly ?

### ISSUE THREAD

**Issue by guiyang882** [HolmesShuan/SNNs-Self-Normalizing-Neural-Networks-Caffe-Reimplementation#7]

**Comment by HolmesShuan** [comment]
README.md `How to use?`

**Comment by guiyang882** [comment]
selu activation function needs to cooperate with the convolutional data initialization parameters. However, the existing caffe parameter initialization method does not seem to meet the input requirements of selu.

**Comment by HolmesShuan** [comment]
https://github.com/bioinf-jku/SNNs/blob/master/Keras-CNN/CIFAR10-Conv-SELU.py
Also, https://keras.io/initializers/ for `lecun_normal`.

**Comment by guiyang882** [comment]
thank you for you reply.
**lecun_normal**, I guess I should implement this method in the caffe framework.
thx !

**Comment by HolmesShuan** [comment]
Tips:
```
weight_filler {
      type: "gaussian"
      std: xxxx # sqrt(1 / fan_in) 
    }
```
`fan_in` could be precomputed for fixed input data size.
